src/components/common/Footer.tsx

```diff
@@ -0,0 +1,95 @@
+import { FC } from 'react';
+import Link from 'next/link';
+
+import { Logo } from './Logo';
+import { Icon } from './Icon';
+import { isExternalUrl } from '../../utils/helpers';
+import config from '../../config';
+import * as routes from '../../routes';
+
+const content = {
+  note: (
+    <p className="mt-2">
+      Made with 💜 by{' '}
+      <Link href={config.raydiantUrl}>
+        <a
+          className="hover:text-slate-700 dark:hover:text-slate-300"
+          target="_blank"
+          rel="noreferrer"
+        >
+          Raydiant
+        </a>
+      </Link>
+    </p>
+  ),
+  menus: [
+    {
+      // TODO: Populate
```

Comment:
We will eventually want to populate this section with links to other significant doc pages.

Card to track that work here: https://trello.com/c/6wiUCVvl/4345-developer-docs-site-populate-footer-with-content

<img width="342" alt="image" src="https://user-images.githubusercontent.com/1712435/174206939-e78b2a3b-362d-4e3e-8493-4809ee337ade.png">

Reply:
Thinking it makes sense to us to link to Raydiant.com dash.raydiant and developers.raydiant somewhere in here. What are your thoughts? Perhaps a new heading "Raydiant Suite" or similar?

Reply:
I'll put some thought into the footer, don't worry about it for now